spirv_reflect.c

```diff
@@ -645,6 +647,32 @@ static SpvReflectResult ParseNodes(Parser* p_parser)
         if (p_node->word_count >= 4) {
           CHECKED_READU32(p_parser, p_node->word_offset + 3, p_parser->source_file_id);
         }
+        if (p_node->word_count >= 5) {
+          const char* p_source = (const char*)(p_parser->spirv_code + p_node->word_offset + 4);
+          char* p_source_temp = (char*)calloc(strlen(p_source), sizeof(char*));
```

Comment:
Is it possible for `p_source` to point to an empty string resulting in `strlen` returning 0?

Reply:
I'm not 100% sure. I assumed that if there is a 5th word, then the source code string exists. However, I think I am missing a +1 here.

Reply:
IIRC `strcpy` below will copy with the null terminator?

Reply:
+1 added

Reply:
Looks good, Thanks.